pkg/agent/handler/agentresource.go

```diff
@@ -2,18 +2,23 @@ package handler
 
 import (
 	"context"
+	"strings"
 	"time"
 
+	agentcache "github.com/Axway/agent-sdk/pkg/agent/cache"
```

Comment:
rather than importing the cache package, can you create a local interface with just the methods you need from the manager?